src/System.Runtime/tests/System/DateTimeOffsetTests.cs

```diff
@@ -871,6 +871,16 @@ public static void ParseExact_String_String_FormatProvider_DateTimeStyles_O()
         }
 
         [Fact]
+        public static void ParseExact_String_String_FormatProvider_DateTimeStyles_R()
+        {
+            DateTimeOffset expected = DateTimeOffset.MaxValue;
+            string expectedString = expected.ToString("r");
+
+            DateTimeOffset result = DateTimeOffset.ParseExact(expectedString, "r", null, DateTimeStyles.None);
+            Assert.Equal(expectedString, result.ToString("r"));
+        }
```

Comment:
Is there a reason we rushed the coreclr change in without having solid tests in place?  I would have much preferred it if we took the extra time to add the tests we wanted for the cases being modified, before merging the changes into coreclr.  It's one thing to have not-great-tests in place for something, but it's another to have not-great-tests in place for something AND make fundamental changes to the implementation.  It should not take very long to add such tests.